tests/sles4sap/monitoring_services.pm

```diff
@@ -124,8 +123,7 @@ sub configure_sap_host_exporter {
     upload_logs("$exporter_config", failok => 1);
 
     # Get the IP port and start exporter
-    my ($exporter_port) = script_output("awk '/port:/ { print \$NF }' $exporter_config", proceed_on_failure => 1) =~ /(\d+)/;
-    $exporter_port ||= 9680;
+    my ($exporter_port) = script_output("awk '/port:/ { print \$NF }' $exporter_config", proceed_on_failure => 1) =~ /^\s*(\d+)\s*$/ ? $1 : 9680;
```

Comment:
That's weird. I thought I removed this. It was to count out spaces but there's no need for that - split is done on spaces by default so there wouldn't be any.

It will work the same - but it's an extra and confusing difference. Let me change it